src/components/expanded-state/swap-settings/MaxToleranceInput.tsx

```diff
@@ -60,7 +68,8 @@ export const MaxToleranceInput = forwardRef(
         slippageRef?.current?.blur();
       },
       reset: () => {
-        onSlippageChange(1);
+        let slippage = getDefaultSlippageFromConfig(currentNetwork);
```

Comment:
maybe `const` instead?

Reply:
we need to transform the slippage in bips to decimal numbers, this one is setting the slippage to 100, we were setting it to 1 before

Reply:
i see we do slippageInBips / 100 in other places